src/main/java/duke/helper/DatetimeHelper.java

```diff
@@ -0,0 +1,22 @@
+package duke.helper;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DatetimeHelper {
+
+  public static String inputFormat = "yyyy-MM-dd HHmm";
+  public static String exampleDatetime = "2015-09-15 1300";
```

Comment:
inputFormat and exampleDatetime should be made final constants since they are not meant to change.